content/operate/rc/security/access-control/data-access-control/create-assign-users.md

```diff
@@ -42,7 +42,9 @@ An error occurs if a user tries to connect to a memcached database with the user
 
     {{<image filename="images/rc/data-access-control-users-add-role.png" width="300px" alt="User select role." >}}
 
-1. Enter and confirm the user's password, then select the check mark to save the user. 
+1. Enter and confirm the user's password. ACL user passwords must be between 8 and 28 characters long and contain a combination of uppercase and lowercase letters, numbers, and special characters. 
```

Comment:
Is the list of supported special characters mentioned elsewhere? If not, it might be a good idea to include it here.

Reply:
```suggestion
1. Enter and confirm the user's password. ACL user passwords must be between 8 and 128 characters long.
```
Thanks for the feedback - I also noticed a typo I made and just decided to remove that part.